diff --git a/adev/src/app/features/home/components/views/build-text.ts b/adev/src/app/features/home/components/views/build-text.ts
index 13f3328905..7e255b216d 100644
--- a/adev/src/app/features/home/components/views/build-text.ts
+++ b/adev/src/app/features/home/components/views/build-text.ts
@@ -48,7 +48,7 @@ export class BuildText extends View {
    * Initialize child views.
    */
   override async init(): Promise<void> {
-    const texture = await loadTexture(this.gl, 'assets/textures/build-msdf.png');
+    const texture = await loadTexture(this.gl, 'assets/textures/construir-para-todos.png');
     texture.minFilter = this.gl.LINEAR;
     texture.generateMipmaps = false;
 
